app/serializers/institution_profile_serializer.rb

```diff
@@ -92,6 +92,7 @@ class InstitutionProfileSerializer < ActiveModel::Serializer
   attribute :preferred_provider
   attribute :stem_indicator
   attribute :facility_map
+  attribute :school_certifying_officials
```

Comment:
You will want to specify which attributes you want returned by creating a `SchoolCertifyingOfficialSerializer`. 
e.g. you don't need `id`, `facility_code`, `institution_name`, etc
Follow the implmentation of the `yellow_ribbon_programs` attribute/serializer as an example.